### PR TITLE
Edit for consistency, clarity, grammar, style

### PR DESCRIPTION
**Pull request by WWitman**
This PR includes:

- Addition of contextual content where needed for clarity
- Structural changes as needed for clarity and consistency
- Grammar, spelling, and other edits

Signed-off-by: Will Witman willwitman@google.com

### CODE REVIEW

File: content/docs/tasks/traffic-management/circuit-breaking/index.md [hunk 1]
```diff
@@ -1,39 +1,51 @@
...
 
     {{< text bash >}}
     $ kubectl apply -f <(istioctl kube-inject -f @samples/httpbin/httpbin.yaml@)
     {{< /text >}}
 
-## Circuit breaker
+    The `httpbin` application serves as the backend service for this task.
 
-Let's set up a scenario to demonstrate the circuit-breaking capabilities of Istio. We should have the `httpbin` service running from the previous section.
+## About this task
```
Comment (geeknoid) [inline content/docs/tasks/traffic-management/circuit-breaking/index.md]: I've mentioned this in another PR this morning, but I don't like this pattern of adding an About this task section. The text for this section should be at the top of the page before anything else. Why would you want someone reading "Before you begin" before they know what the task is about?

File: content/docs/tasks/traffic-management/circuit-breaking/index.md [hunk 2]
```diff
@@ -1,39 +1,51 @@
...
-1.  Create a [destination rule](/docs/reference/config/istio.networking.v1alpha3/#DestinationRule) to specify our circuit breaking settings when calling the `httpbin` service:
+Circuit breaking is an important pattern for creating resilient microservice
+applications. Circuit breaking allows you to write applications that limit the impact of failures, latency spikes, and other undesirable effects of network peculiarities.
+
+In this task, you will configure circuit breaking rules and then test the
+configuration by intentionally "tripping" the circuit breaker.
+
+## Configuring the circuit breaker
+
+Before continuing, be sure the `httpbin` service is running in your mesh, as
```
Comment (geeknoid) [inline content/docs/tasks/traffic-management/circuit-breaking/index.md]: Why repeat this? There's already a before you begin section for this...

File: content/docs/tasks/traffic-management/circuit-breaking/index.md [hunk 3]
```diff
@@ -83,160 +95,181 @@ Let's set up a scenario to demonstrate the circuit-breaking capabilities of Isti
...
-We see `12` for the `upstream_rq_pending_overflow` value which means `12` calls so far have been flagged for circuit breaking.
+## Adding a client
+
+Create a client to send traffic to the `httpbin` service. The client is
+a simple load-testing client called [fortio](https://github.com/istio/fortio).
+Fortio lets you control the number of connections, concurrency, and
+delays for outgoing HTTP calls. You will use this client to "trip" the circuit breaker
+policies you set in the `DestinationRule`. 
+
+1. Inject the client with the istio sidecar proxy so network interactions are
```
Comment (geeknoid) [inline content/docs/tasks/traffic-management/circuit-breaking/index.md]: istio -> Istio

File: content/docs/tasks/traffic-management/fault-injection/index.md [hunk 4]
```diff
@@ -71,56 +78,79 @@ continue without any errors.
             subset: v1
     {{< /text >}}
 
-    Allow several seconds to account for rule propagation delay to all pods.
+    Allow several seconds to for the new rule to propagate to all pods.
```
Comment (geeknoid) [inline content/docs/tasks/traffic-management/fault-injection/index.md]: to for -> for

File: content/docs/tasks/traffic-management/fault-injection/index.md [hunk 5]
```diff
@@ -71,56 +78,79 @@ continue without any errors.
...
-    to load within approximately 7 seconds. To see the web page response times, open the
-    *Developer Tools* menu in IE, Chrome or Firefox (typically, key combination _Ctrl+Shift+I_
-    or _Alt+Cmd+I_), tab Network, and reload the `productpage` web page.
+    You expect the Bookinfo home page to load without errors in approximately
+    7 seconds. However, there is a problem: the Reviews section displays an error
+    message:
 
-    You will see that the webpage loads in about 6 seconds. The reviews section will show
-    *Sorry, product reviews are currently unavailable for this book*.
+     {{< text code >}}
```
Comment (geeknoid) [inline content/docs/tasks/traffic-management/fault-injection/index.md]: code -> plain

**Comment by WWitman** [comment]
@geeknoid I rebased the branch and it's good to merge now. Thanks to @rcaballeromx for the help.

**Pull request merged**